src/api/apps.ts

```diff
@@ -102,6 +104,33 @@ export const patchSettings = (args: {
     .then(({ data }) => data);
 };
 
+export const getAppActions = async (args: { token: string; itemId: string; apiHost: string }) => {
+  const { token, itemId, apiHost } = args;
+  return axios
+    .get(`${apiHost}/${buildGetAppActionsRoute(itemId)}`, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    })
+    .then(({ data }) => data);
+};
+
+export const postAppAction = (args: {
+  token: string;
+  itemId: string;
+  body: any;
+  apiHost: string;
+}) => {
+  const { token, itemId, apiHost, body } = args;
+  return axios
+    .post(`${apiHost}/${buildPostAppActionsRoute({ itemId })}`, body, {
+      responseType: 'blob',
```

Comment:
You don't need this.

Reply:
Which line(s)? I don't understand what is useless here.